src/networkProviders/interface.ts

```diff
@@ -143,22 +147,7 @@ export interface IAddress {
     bech32(): string;
 }
 
-export interface ITransactionNext {
-    sender: string;
-    receiver: string;
-    gasLimit: bigint;
-    chainID: string;
-    nonce: bigint;
-    value: bigint;
-    senderUsername: string;
-    receiverUsername: string;
-    gasPrice: bigint;
-    data: Uint8Array;
-    version: number;
-    options: number;
-    guardian: string;
-    signature: Uint8Array;
-    guardianSignature: Uint8Array;
-    relayer?: string;
-    innerTransactions?: ITransactionNext[];
-}
+/**
+ * @deprecated This will be remove with the next release (replaced by the `ITransaction` interface from "src/interface.ts").
+ */
+export type ITransactionNext = ITransactionAsInSpecs;
```

Comment:
Previously, `ITransactionNext` was almost the same as `ITransactionAsInSpecs`, with the exception of:
 - `ITransactionNext.relayer` was optional, `ITransactionAsInSpecs.relayer` isn't (but empty is allowed)
 - `ITransactionNext.innerTransactions` was optional, `ITransactionAsInSpecs.innerTransactions` isn't (but empty is allowed).

This should not be considered a breaking change, since it's an alteration of an interface that's merely used as the substitute of `Transaction within the network provider components newly moved to `src/networkProviders`.